.circleci/config.yml

```diff
@@ -798,7 +814,11 @@ workflows:
       - node-mongoose: *matrix-supported-node-versions
       - node-mysql: *matrix-supported-node-versions
       - node-net: *matrix-supported-node-versions
-      - node-next: *matrix-supported-node-versions
+      # Next consistently fails on Node.js 17 for strange OpenSSL reasons.
```

Comment:
Does it still fail with `NODE_OPTIONS=--openssl-legacy-provider`?

Reply:
Trying that in latest push.